apps/cowswap-frontend/src/modules/accountProxy/pure/AccountCard/IdentityIcon.tsx

```diff
@@ -0,0 +1,34 @@
+import { ReactNode } from 'react'
+
+import { UI } from '@cowprotocol/ui'
+
+import SVG from 'react-inlinesvg'
+import styled from 'styled-components/macro'
+
+const IdentityIconStyled = styled.div`
+  width: 70px;
+  height: 70px;
+  border-radius: 70px;
+  background: var(${UI.COLOR_TEXT_OPACITY_10});
+  display: flex;
+  align-items: center;
+  justify-content: center;
+
+  > svg,
+  > img {
+    width: 100%;
+    height: 100%;
+    object-fit: contain;
+    padding: 25%;
+    fill: var(${UI.COLOR_TEXT_OPACITY_70});
+  }
+`
+
+interface IdentityIconProps {
+  icon?: string
+  className?: string
+}
+
+export function IdentityIcon({ icon, className }: IdentityIconProps): ReactNode {
```

Comment:
This component is not used in `AccountCard, but in `WalletNotConnected` component. So it should be defined in `pure/IdentityIcon/index.tsx` or `pure/WalletNotConnected/IdentityIcon.tsx`

Reply:
Ah yeah, planned to use the same component for all cards, but in the end didn't do it. I'll simplify so we don't need the separate file (just integrate in WalletNotConnected).